src/parser/cssParser.ts

```diff
@@ -1172,14 +1172,33 @@ export class Parser {
 				return this.finish(node, ParseError.IdentifierExpected);
 			}
 			if (!this.hasWhitespace() && this.accept(TokenType.ParenthesisL)) {
-				let tryAsSelector = () => {
-					let selector = this._parseSimpleSelector();
-					if (selector && this.peek(TokenType.ParenthesisR)) {
-						return selector;
+				let tryAsSelectors = () => {
+					const firstSelector = this._parseSimpleSelector();
+					if (!firstSelector) {
+						return null;
+					} else {
+						const selectors = [firstSelector];
```

Comment:
Thanks for the suggestion. I actually went one step further for `_parseSelector`, since Safari (which has full support for :matches) seems to allow selector + combinators as args for :matches.

I'm wondering why you were suggesting using `_parsePageSelector` here, though.

Reply:
Ah, I meant _parseSimpleSelector.
The spec is really complicated on that one. https://drafts.csswg.org/selectors-4/#selector-list.
Now that I read it and that I see how far it is from being a standard I wonder if we should support this. I general I postpone such requests as it's too much work.

Reply:
Yeah it's still in Working Draft status and constantly being updated:

![image](https://user-images.githubusercontent.com/4033249/35291702-025df5f2-0023-11e8-9b08-11627aeeb4d2.png)

Let's postpone it until it becomes at least a Candidate Recommendation.

Or per our discussion, we can try to have a separate postcss validator that's based on postcss parser.